tests/suites/test_suite_md.function

```diff
@@ -294,8 +294,8 @@ void mbedtls_md_hmac( char * text_md_name, int trunc_size,
 /* END_CASE */
 
 /* BEGIN_CASE */
-void md_hmac_multi( char * text_md_name, int trunc_size, data_t * key_str,
-                    data_t * src_str, data_t * hex_hash_string )
+void md_hmac_multi( char *text_md_name, data_t *key_str,
```

Comment:
Same thing here, `int trunc_size` declaration was removed but is used in the function body

Reply:
Removing the argument seems OK but it was done in the wrong commit, please fix.